crates/bevy_pbr/src/render/parallax_mapping.wgsl

```diff
@@ -0,0 +1,102 @@
+#define_import_path bevy_pbr::parallax_mapping
```

Comment:
Note, this relies on the `height_map_texture` and `height_map_sampler` to be in scope, which requires `#import`ing the `pbr_bindings` wgsl file